espeon/command/install_dependencies.lua

```diff
@@ -10,18 +12,18 @@ return {
     if platform == 'linux' then
       exec({
         'sudo apt-get install screen',
-        'sudo apt-get install python-pip',
-        'sudo -H pip install esptool --upgrade',
+        'sudo apt-get install python-pip || sudo apt-get install python3-pip',
+        'sudo -H pip install esptool --upgrade || sudo -H pip3 install esptool --upgrade',
```

Comment:
Accounts for Ubuntu 20.04 and any other Debian-based distro that doesn't allow bare `python` or `pip` anymore.